test/tools/dshell_prebuilt/dshell_prebuilt.d

```diff
@@ -69,6 +73,11 @@ void dshellPrebuiltInit(string testDir, string testName)
         mixin(`Vars.set("` ~ var ~ `", requireEnv("` ~ var ~ `"));`);
     }
 
+    static foreach (var; optionalEnvVars)
```

Comment:
awwwww why that?

Reply:
Copy & Paste error. I'll remove the `static`